.github/workflows/coverage.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           coverage xml
           coverage report
 
-      - uses: codecov/codecov-action@v3
+      - uses: codecov/codecov-action@v4
```

Comment:
There were breaking changes in this release!

Reply:
I don't think that the codecov action worked for external contributors previously.